app/konnect/dev-portal/applications/dev-apps.md

```diff
@@ -7,20 +7,40 @@ content_type: how-to
 
 ## Create an application
 
-To create a new application, log in to the Dev Portal and click **My Apps**. From the **My Apps** page, follow these instructions: 
+### Using the catalog
 
-1. Click the **New App** button.
+1. Sign in to the Dev Portal and click **Catalog**. Find the API product version you want to register for and click **Register**
 
-2. Fill out the **Create New Application** form with your application name, reference ID, and description.
+1. Select **Create an application**. 
+
+1. Complete the Create New Application form by providing your application's name, unique reference ID, and a brief description. Kong will auto select the appropriate auth strategy for the API product version you're registering for.
```

Comment:
How does Kong know what the appropriate auth strategy is?

Reply:
the auth strategies are set on the api product version so you go to register for the api product version, we just prepopulate it